src/main/webapp/app/core/websocket/websocket.service.ts

```diff
@@ -167,7 +167,14 @@ export class JhiWebsocketService implements OnDestroy {
                 this.myListeners[channel] = this.createListener(channel);
             }
             this.subscribers[channel] = this.stompClient.subscribe(channel, data => {
-                this.listenerObservers[channel].next(JSON.parse(data.body));
+                const res = JSON.parse(data.body);
+                if (!res.error) {
```

Comment:
I think it's a good idea to do the check here, not in the components.
This way we can use subscribe(res => handleRes(), err => handleError()).
Otherwise we would have to do subscribe (res => !res.error? handleRes() : handleError()).